stratum/lib/security/credentials_manager.cc

```diff
@@ -14,53 +14,86 @@
 #include "stratum/lib/macros.h"
 #include "stratum/lib/utils.h"
 
-DEFINE_string(ca_cert_file, "", "CA certificate path");
-DEFINE_string(server_key_file, "", "gRPC Server private key path");
-DEFINE_string(server_cert_file, "", "gRPC Server certificate path");
+DEFINE_string(ca_cert_file, "", "Path to CA certificate file");
+DEFINE_string(server_key_file, "", "Path to gRPC server private key file");
+DEFINE_string(server_cert_file, "", "Path to gRPC server certificate file");
+DEFINE_string(client_key_file, "", "Path to gRPC client key file");
+DEFINE_string(client_cert_file, "", "Path to gRPC client certificate file");
```

Comment:
Could probably merged into one. Having client and server in one binary is unlikely? But I like the `client/server` distinction in the flag name.

Reply:
Updating the test code showed that the separation is useful.